src/cascade/model/integrands.py

```diff
@@ -7,20 +7,24 @@
 CODELOG, MATHLOG = getLoggers(__name__)
 
 
-def make_average_integrand_cases_from_gbd(execution_context, sexes):
+def make_average_integrand_cases_from_gbd(execution_context, sexes, include_birth_prevalence=False):
+    """
+    """
     gbd_age_groups = get_age_groups(execution_context)
     age_ranges = [(r.age_group_years_start, r.age_group_years_end) for _, r in gbd_age_groups.iterrows()]
     time_ranges = [(y, y) for y in get_years(execution_context)]
 
+    # Assuming using the first set of weights, which is constant.
+    weight_id = 0
```

Comment:
We do not have our act together regarding weights. This is not the PR in which to fix that.

Reply:
Yep, this is the third of fourth time I've carried this forward without really understanding it.